src/modules/imageresizer/ui/ImageResizerUI.csproj

```diff
@@ -66,13 +66,13 @@
     </PackageReference>
     <PackageReference Include="Microsoft.Xaml.Behaviors.Wpf" Version="1.1.19" />
     <PackageReference Include="ModernWpfUI" Version="0.9.4" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="System.IO.Abstractions">
       <Version>12.2.5</Version>
     </PackageReference>
+    <PackageReference Include="System.Text.Json" Version="5.0.2" />

```

Comment:
This dll should be added to the installer.

Reply:
What is the reason we use a package here?

Reply:
System.Text.Json version 5 has a lot of improvements over 3 which we need.
PT Run also has a dependency to V5 Added, so it makes sense to keep that the same across multiple modules.

Reply:
Okay, but we still need to add a dll to the installer.